diff --git a/src/classes/presentation_controller.vala b/src/classes/presentation_controller.vala
index a231c67..1593fae 100644
--- a/src/classes/presentation_controller.vala
+++ b/src/classes/presentation_controller.vala
@@ -379,7 +379,7 @@ namespace pdfpc {
 
                     // Assuming we're in the middle of the current slide
                     double expected_progress =
-                        (this.current_user_slide_number + 0.5)/(this.metadata.get_end_user_slide() + 1);
+                        (this.current_slide_number + 0.5)/(this.metadata.user_slide_to_real_slide(this.metadata.get_end_user_slide(), true) + 1);
 
                     int expected_time = (int) (duration*expected_progress);
 
