diff --git a/src/Nixfmt/Pretty.hs b/src/Nixfmt/Pretty.hs
index 4f99661..b90fddd 100644
--- a/src/Nixfmt/Pretty.hs
+++ b/src/Nixfmt/Pretty.hs
@@ -349,14 +349,7 @@ instance Pretty Parameter where
         -- If the braces are on different lines, keep them like that
         if sourceLine bopen /= sourceLine bclose
           then hardline
-          else case attrs of
-            [ParamEllipsis _] -> line
-            -- Attributes must be without default
-            [ParamAttr _ Nothing _] -> line
-            [ParamAttr _ Nothing _, ParamEllipsis _] -> line
-            [ParamAttr _ Nothing _, ParamAttr _ Nothing _] -> line
-            [ParamAttr _ Nothing _, ParamAttr _ Nothing _, ParamEllipsis _] -> line
-            _ -> hardline
+          else line
   pretty (ContextParameter param1 at param2) =
     pretty param1 <> pretty at <> pretty param2
 
@@ -720,9 +713,9 @@ instance Pretty Expression where
   -- Attrset parameter
   pretty (Abstraction param colon (Term t))
     | isAbsorbable t =
-        pretty param <> pretty colon <> line <> group (prettyTermWide t)
+        pretty param <> pretty colon <> hardspace <> group (prettyTermWide t)
   pretty (Abstraction param colon body) =
-    pretty param <> pretty colon <> line <> pretty body
+    pretty param <> pretty colon <> hardspace <> pretty body
   pretty (Application f a) =
     prettyApp False mempty False f a
   -- not chainable binary operators: <, >, <=, >=, ==, !=
