diff --git a/lua/onedark/highlights.lua b/lua/onedark/highlights.lua
index 76c3299..ec5abff 100644
--- a/lua/onedark/highlights.lua
+++ b/lua/onedark/highlights.lua
@@ -241,8 +241,6 @@ if vim.api.nvim_call_function("has", { "nvim-0.8" }) == 1 then
         -- Misc
         ["@none"] = colors.Fg,
         ["@conceal"] = colors.Grey,
-        ["@nospell"] = colors.Fg,
-        ["@spell"] = colors.Fg,
 
         -- Operators
         ["@operator"] = colors.Fg,
